### ISSUE TITLE
Failed to render instance of RecordView

### ISSUE THREAD

**Issue by benjam1n** [Clipy/KeyHolder#6]
Hi guys!

I have installed KeyHolder via CocoaPods:
```
...
pod 'Magnet'
pod 'KeyHolder'
```

But there are some weird errors in Xcode:
![screen shot 2017-05-19 at 13 02 27](https://cloud.githubusercontent.com/assets/103969/26245267/9d3fde9a-3c93-11e7-89a4-3e56db7d9624.png)
![screen shot 2017-05-19 at 13 02 18](https://cloud.githubusercontent.com/assets/103969/26245268/9d49f484-3c93-11e7-835a-ea85db690c6f.png)
![screen shot 2017-05-19 at 13 01 53](https://cloud.githubusercontent.com/assets/103969/26245269/9d4ac080-3c93-11e7-92df-4e9588628f62.png)

What's especially confusing is this line:
![screen shot 2017-05-19 at 13 04 34](https://cloud.githubusercontent.com/assets/103969/26245293/bd3030b0-3c93-11e7-9196-82885d351668.png)

What have I done wrong? :)

ps. is there an easy way to change components font size?

**Comment by Econa77** [comment]
@benjam1n I could reproduce the problem that IBDesignable error. I will fix it. However, this error does not cause the application to crash.

Specifying "com.clipy-app.KeyHolder" is Carthage or special correspondence when manual installation.

Since the font size of the component depends on the height of the RecordView, it is not possible to change the font size.
Do you want to change the font size directly?

**Comment by benjam1n** [comment]
@Econa77 Yes, indeed it does not crash but it's still weird since in interface builder it shows for like 1s (as green button) and then disappears. So if you can fix it, it would be awesome. I mean this + magnet is like the best shortcut component for macOS. You guys already saved me a couple days of work. :)

About the font size, yeah I was thinking about how could I subclass it or something to make font size fixed and change the text, let's say when its focused and recording I want to just have "Recording..." and so on. Could you point me in some direction?

Thanks a lot!

**Comment by Econa77** [comment]
@benjam1n Thanks :)
I checked the error of IBDesignable and found out that it is a link error of the framework using CocoaPods. ( ref: https://github.com/CocoaPods/CocoaPods/issues/5334 )

The workaround is as follows, but the fundamental modification has to wait for modification of CocoaPods or Xcode.
https://github.com/CocoaPods/CocoaPods/issues/5334#issuecomment-279994132

In the current layout there is an arrangement of modifier keys, so changing the font size may be tough. However, I think that the display in the 「Recording...」 text is a good idea, so I will implement it.
In addition, I am trying to create a small view component (KeyHolder-mini) like [ShortcutRecorder](https://github.com/Kentzo/ShortcutRecorder), so please wait implementation.

**Comment by jasonm23** [comment]
I also saw this error, the only reasonable workaround I can think of is to install KeyHolder as a git submodule until CocoaPods/CocoaPods#5334 (comment) is resolved
